apps/speed.c

```diff
@@ -212,7 +212,8 @@ static int AES_cbc_256_encrypt_loop(void *args);
 static int AES_ige_192_encrypt_loop(void *args);
 static int AES_ige_256_encrypt_loop(void *args);
 static int CRYPTO_gcm128_aad_loop(void *args);
-static int EVP_Update_loop(void *args);
+static int EVP_Encrypt_loop(void *args);
+static int EVP_Digest_loop(void *args);
```

Comment:
Is this a duplicated declaration of `EVP_Digest_loop`?
Should it be `EVP_Decrypt_loop` instead?